ThatJustin/budget-lens
# BUD-128 login option with google displayed on the login page. & BUD-129 Use OAuth already setup in BUD-55 to Login with google

**New Feature:**
1.  login option with google displayed on the login page. 
2. Use OAuth already set up in Login Page to log in with Google.

**To Test It**
Run the project with the emulator ---> click on the budget lens logo under the name ---> Click on the Google logo at the bottom of the Login Page ---> Input your Google Account username and password ---> You will get an alert on your other google device or email ---> Confirm to login ---> You will see a page that shows your google email and your full name ---> Click the bottom Sign out button to Sign out. ( The emulator will remember your google account)



**Still needs to be done**

Store the google account in our own database
